WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderWebView.java

```diff
@@ -78,11 +84,21 @@ public boolean isDestroyed() {
 
     public ReaderWebView(Context context, AttributeSet attrs) {
         super(context, attrs);
+
+        TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.BoundedWidth);
```

Comment:
Perhaps we could avoid obtaining mBoundedWidth both here and in the next constructor by simply calling `this(context, attrs, 0);` here?